users/ignoring.rst

```diff
@@ -98,26 +124,15 @@ The ``.stignore`` file contains a list of file or path patterns. The
    they are preventing directory deletion. This prefix should be used by any OS
    generated files which you are happy to be removed.
 
+   .. note::
+
+      Prefixes can be specified in any order (e.g. ``(?d)(?i)``), but cannot
+      be combined in a single pair of parentheses like :strike:`(?di)`.
+
 -  A line beginning with ``//`` is a comment and has no effect. The same double
```

Comment:
This should be shuffled way up higher IMO, probably close to the `/rooted_ignore_pattern` bullet point.  Then the closing note about prefix order stands correctly *after* the list, but still directly related to the prefixes.